po/ja.po

```diff
@@ -2380,6 +2423,10 @@ msgid ""
 "* Built-in support for testing.\n"
 "* Excellent Language Server Protocol support."
 msgstr ""
+"* 優秀なコンパイルエラー。\n"
+"* 組み込みの依存関係マネージャ。\n"
+"* 組み込みのテストサポート。\n"
+"* 優秀な言語サーバプロトコル（LSP）のサポート。"
```

Comment:
I don't think we translate "Language Server Protocol" into Japanese. You can write "Language Server Protocol (LSP)" as is.